### ISSUE TITLE
Chest progression can be skipped by using logs to craft 4 chests

### ISSUE THREAD

**Issue by notWaterGuy12** [hoklai1997/Coven-Undone#3]
Yep that's about it, noticed a break in the chest progression
Edit: this can also technically be skipped using the cabinet recipe for Rustic's cabinets

**Comment by notWaterGuy12** [comment]
I'm also having trouble getting the Doggy Charm Ritual to work, time is correct but it refuses to work for me

**Comment by hoklai1997** [comment]
are you on the most recent version? also does the recipe for the chests also work or just appear in JEI? I dont think i can make the recipe disappear, however it doesnt actually work if you try it. For the dog charm, im unsure, but if the problem still persists on the most recent version than lmk.

**Comment by notWaterGuy12** [comment]
Yep most recent version, the recipe for chests does not work. The cabinet, however, does still work
